packages/list-reusable-blocks/package.json

```diff
@@ -22,6 +22,7 @@
 	"engines": {
 		"node": ">=12"
 	},
+	"gutenberg:source": "src/index.js",
```

Comment:
Ideally, we should add this to all packages so that we don't have to use `react-native` as the main field.

Or we can write our own custom resolver, not sure if it's worth it though.